static/app/components/modals/explore/saveQueryModal.tsx

```diff
@@ -209,11 +208,6 @@ const StyledButtonBar = styled(ButtonBar)`
   }
 `;
 
-export const modalCss = css`
-  max-width: 700px;
-  margin: 70px auto;
-`;
-
```

Comment:
`modalCss` is usually imported alongside the `Modal` itself, but in this case, it isn’t:

https://github.com/getsentry/sentry/blob/8a0590c7a64540c42d35f09097cfc5b08482c6fd/static/app/actionCreators/modal.tsx#L418-L424

Please let me know if the actual fix is to add `modalCss` to `openModal` 🙏